gradle-plugin/src/main/kotlin/org/mozilla/android/MozillaPlugin.kt

```diff
@@ -6,9 +6,42 @@ package org.mozilla.android
 
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
 
 class MozillaPlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
+        with(project) {
+            // Add a custom repository for application-services dependency resolution.
+            // This should be temporary while https://github.com/mozilla/application-services
+            // isn't publishing to maven.mozilla.org yet.
+            // See https://github.com/mozilla/application-services/issues/252.
+            val customName = "appservices"
+            val customURI = URI.create("https://dl.bintray.com/ncalexander/application-services")
+
+            // If there's already a Maven repo with the right URL, or even the right name, roll with it.
+            // The name gives the opportunity to customize, if it helps in the wild.
+            val existing = project.repositories.find {
+                (it is MavenArtifactRepository) && (it.url == customURI || it.name == customName)
+            }
+
+            // Otherwise, inject the dependency.
+            if (existing == null) {
+                logger.info("Injecting repository for project '${project}': '${customName}' Maven repository with url '${customURI.toASCIIString()}'")
+                val customMavenRepo = project.repositories.maven {
+                    it.name = customName
+                    it.url = customURI
+                }
+
+                project.repositories.removeAt(project.repositories.size - 1)
+                project.repositories.addFirst(customMavenRepo)
+
+                logger.info("Repository list for ${project} after injection:")
```

Comment:
This only runs with `--info`, right? I wouldn't want this to log every time.

Reply:
Yup, only with `--info`.